api/src/apps/routes.rs

```diff
@@ -211,7 +216,7 @@ async fn logs(
             }
         },
     };
-    let limit = limit.unwrap_or(20_000);
+    let limit = limit.unwrap_or_default();
```

Comment:
I understand the mistake, maintaining limit as an `Option` at the infrastructure level instead of unwrapping in `routes` will result in better handling.